docs/src/GettingStarted/Installation.md

```diff
@@ -78,4 +78,8 @@ $ julia --project tutorials/Atmos/dry_rayleigh_benard.jl
 ```
 
 The `ClimateMachine` is CUDA-enabled and will use GPU(s) if available. To run
-on the CPU, set the environment variable `CLIMATEMACHINE_GPU` to `false`.
+on the CPU, set the environment variable `CLIMATEMACHINE_SETTINGS_DISABLE_GPU` to `true` by running
+
+```
+$ export CLIMATEMACHINE_SETTINGS_DISABLE_GPU=true
+```
```

Comment:
Since this won't work in all shell (e.g, fish) might be better to say

> on the CPU, set the environment variable
> `CLIMATEMACHINE_SETTINGS_DISABLE_GPU=true`. This can either be done by done
> inline with the Julia launch command
> ```
> CLIMATEMACHINE_SETTINGS_DISABLE_GPU=true julia --project
> ```
> or for the whole shell session, for example with `bash` this would be
> ```
> export CLIMATEMACHINE_SETTINGS_DISABLE_GPU=true
> ```

(Suggestion wasn't displaying right)

Reply:
Oooh, thank you so much for the great suggestion!